packages/roosterjs-content-model-core/lib/command/paste/cleanHtmlComments.ts

```diff
@@ -1,7 +1,7 @@
 const HtmlCommentStart = '\x3C!--';
 const HtmlCommentStart2 = '<!--';
 const HtmlCommentEnd = '-->';
-const styleTag = '<style';
+const styleTag = '<style>';
```

Comment:
I think we still need to allow any attributes in this tag, so we just need to improve the original check, for example
```ts
const styleTag = '<style'
...
const index = ...
const nextChar = html.substring(index + styleTag.length, 1);

if (/\W/.test(nextChar)) { ...
```